src/main/java/io/elastic/api/SelectModelProvider.java

```diff
@@ -0,0 +1,23 @@
+package io.elastic.api;
+
+import com.google.gson.JsonObject;
+
+
+/**
+ * 
+ * Interface to be implemented by components which want to provide select model
```

Comment:
The 1st sentence must end with a dot. Otherwise Javadoc generator fails with an error.